networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 You can install the AWS Load Balancer Operator on the Secure Token Service (STS) cluster.
 
-The AWS Load Balancer Operator relies on `CredentialsRequest` to bootstrap the Operator and for each `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available. The Cloud Credential Operator does not provision the secrets automatically in the STS cluster. You must set the credentials secrets manually by using the `ccoctl` binary. 
+The AWS Load Balancer Operator relies on `CredentialsRequest` to bootstrap the Operator and for each `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available. The Cloud Credential Operator does not provision the secrets automatically in the STS cluster. You must set the credentials secrets manually by using the `ccoctl` binary.
```

Comment:
The sentence "The AWS Load Balancer Operator waits until the required secrets are created and available." is passive voice. The problem is the phrase "are created". A possible rewrite could be: "The AWS Load Balancer Operator requires the secrets to be created and available for correct function."

Reply:
See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs under the heading "Voice" . (Please ping me if you have trouble seeing the IBM style guide.)